src/containers/Transactions/SimpleTab.jsx

```diff
@@ -31,38 +32,23 @@ class SimpleTab extends Component {
     const { t } = this.props
     return (
       <>
-        <div className="row">
-          <div className="label">
-            {t('formatted_date', { timeZone: TIME_ZONE })}
-          </div>
-          <div className="value">{time}</div>
-        </div>
-        <div className="row">
-          <div className="label">{t('ledger_index')}</div>
-          <div className="value">
-            <Link to={`/ledgers/${ledgerIndex}`}>{ledgerIndex}</Link>
-          </div>
-        </div>
-        <div className="row">
-          <div className="label">{t('account')}</div>
-          <div className="value account">
-            <Account account={account} />
-          </div>
-        </div>
-        <div className="row">
-          <div className="label">{t('sequence_number')}</div>
-          <div className="value">
```

Comment:
What were thte `value` classes doing before? (I don't see an equivalent ported over)

Reply:
The SimpleRow component has that.  Whatever children elements you supply get put into value.